packages/react/src/components/form/index.tsx

```diff
@@ -1,2 +1,5 @@
 export { Form } from "./form";
 export type { Props as FormProps } from "./form";
+
+// does not work as Form.Error
+export { Error as FormErrorState } from "./error";
```

Comment:
I tried to export this as part of the `Form` namespace but this results with build errors. Vite can't resolve these modules.